en/user_guide/global_page_blocks.rst

```diff
@@ -0,0 +1,23 @@
+Global Pages and Blocks
+=======================
+
+In Open Orchestra, one of the purposes is to simplify the creation of pages by using configurable blocks.
+These blocks must be configured according to their functionnality (see `Block creation`_ and `Block list`_).
```

Comment:
The second sentence is a little repetition I think, you should just put the link for the block creation and list doc